hazelcast/src/main/java/com/hazelcast/config/RingbufferConfig.java

```diff
@@ -88,6 +88,7 @@ public RingbufferConfig(RingbufferConfig config) {
         this.asyncBackupCount = config.asyncBackupCount;
         this.timeToLiveSeconds = config.timeToLiveSeconds;
         this.inMemoryFormat = config.inMemoryFormat;
+        this.ringbufferStoreConfig = config.ringbufferStoreConfig;
```

Comment:
That's a good one. Didn't think about it, since we normally just use that constructor to create the read-only configuration. But it's public, so we should make it a proper clone.